api/src/test/java/org/pmiops/workbench/actionaudit/targetproperties/WorkspaceTargetPropertyTest.java

```diff
@@ -0,0 +1,146 @@
+package org.pmiops.workbench.actionaudit.targetproperties;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.pmiops.workbench.access.AccessTierService;
+import org.pmiops.workbench.model.ResearchPurpose;
+import org.pmiops.workbench.model.SpecificPopulationEnum;
+import org.pmiops.workbench.model.Workspace;
+
+public class WorkspaceTargetPropertyTest {
+  private Workspace workspace1;
+  private Workspace workspace2;
+
+  @BeforeEach
+  public void setUp() {
+    ResearchPurpose researchPurposeAllFieldsPopulated =
+        new ResearchPurpose()
+            .intendedStudy("stubbed toes")
```

Comment:
not sure why it changed the order of these, but I confirmed the list is equivalent